reloj-checador/app/views/employees/index.html.erb

```diff
@@ -3,6 +3,48 @@
 <div class="container">
   <h1 class="title">Employees</h1>
   <div class="content">
+<<<<<<< HEAD
+    <%= link_to 'New Employee', new_employee_path, :class => "btn btn-primary" %>
+    <table class="table table-striped table-responsive mt-4">
+      <thead>
+        <tr>
+          <th scope="col">Email</th>
+          <th scope="col">Name</th>
+          <th scope="col">Lastname</th>
+          <th scope="col">Position</th>
+          <th scope="col">Private number</th>
+          <th scope="col">Active</th>
+          <th scope="col">Company</th>
+          <th scope="col">Actions</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @employees.each do |employee| %>
+          <tr>
+            <td scope="row"><%= employee.email %></td>
+            <td><%= employee.name %></td>
+            <td><%= employee.lastname %></td>
+            <td><%= employee.position %></td>
+            <td><%= employee.private_number %></td>
+            <td><%= employee.active %></td>
+            <td><%= employee.company_id %></td>
+            <td class="button-td">
+              <%= link_to employee, :class => "btn btn-success" do %>
+                <i class="fas fa-eye"></i>show 
+              <% end %>
+              <%= link_to edit_employee_path(employee), :class => "btn btn-warning" do%>
+                <i class="fas fa-user-edit"></i>Edit
+              <% end %>
+               <%= link_to employee, method: :delete, data: { confirm: 'Are you sure?' }, :class => "btn btn-danger" do %>
+                <i class="fas fa-trash"></i>Destroy
+              <% end %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+=======
```

Comment:
I notice that here a conflict occurred between the branches and the master and the lines indicating the conflicting lines were not removed.

```
<<<<<<< HEAD

 =======

>>>>>>> 2180a79516cefc963819e448676c43d41cf92aa3
```

Reply:
Omg! it slipped my mind jajaja thanks